nengo_gui/guibackend.py

```diff
@@ -166,14 +166,19 @@ def browse(self):
 
         for f in sorted(os.listdir(path)):
             ff = os.path.join(path, f)
+            ff = ff.encode('utf-8')
```

Comment:
I'd prefer to have this in one line by appending the `.encode('utf-8')` to the previous line.